db/schema.rb

```diff
@@ -640,14 +664,14 @@
   add_foreign_key "learning_hour_types", "casa_orgs"
   add_foreign_key "learning_hours", "learning_hour_types"
   add_foreign_key "learning_hours", "users"
-  add_foreign_key "mileage_rates", "casa_orgs"
+  add_foreign_key "mileage_rates", "casa_orgs", validate: false
   add_foreign_key "mileage_rates", "users"
-  add_foreign_key "notes", "users", column: "creator_id"
+  add_foreign_key "notes", "users", column: "creator_id", validate: false
   add_foreign_key "other_duties", "users", column: "creator_id"
   add_foreign_key "patch_notes", "patch_note_groups"
   add_foreign_key "patch_notes", "patch_note_types"
   add_foreign_key "placement_types", "casa_orgs"
-  add_foreign_key "placements", "casa_cases"
+  add_foreign_key "placements", "casa_cases", validate: false
```

Comment:
Let's manually take these out.

Reply:
There's a ticket that I'll write up to address why this is happening.

Reply:
To clarify. If notes is going away, does that mean that it should be rolled in as a contact topic?

Reply:
Actually, run the commands to rebuild it and they will go away.

Reply:
it did. btw @schoork you have anything against me rebasing the branch and squashing some commits? Github is gonna lose of the changes since last review but I don't really like that I have merges between some of my commits.

Reply:
No that's fine